src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -932,6 +932,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Private ReadOnly Property Conversion As Conversion
+            Get
+                Return New Conversion(Me.ConversionKind)
+            End Get
+        End Property
+
+        Private ReadOnly Property IConversionExpression_IsIdentity As Boolean Implements Semantics.IConversionExpression.IsIdentity
+            Get
+                Return Me.Conversion.IsIdentity
```

Comment:
Conversion structure is not used by VB compiler itself, it is created only as an output for external consumers. We should keep it this way. All the answers could be answered directly off of Me.ConversionKind, see how Conversion structure does this, for example. There is no need to add an additional overhead.